main.go

```diff
@@ -100,6 +100,15 @@ func main() {
 	}
 	defer utils.Close(lf)
 
+	endpoint := os.Getenv("SPANNER_API_ENDPOINT")
+	emulator_host := os.Getenv("SPANNER_EMULATOR_HOST")
+	if !((endpoint == "" && (emulator_host == "" || emulator_host == "localhost:9010")) || (endpoint == "staging-wrenchworks.sandbox.googleapis.com:443" && emulator_host == "")) {
+		fmt.Printf("\nWarning: Endpoint specified may be incorrect: \n" +
+			"For connecting to prod set SPANNER_API_ENDPOINT as blank \n" +
+			"For connecting to emulator set SPANNER_EMULATOR_HOST=localhost:9010 \n" +
+			"For connecting to staging set SPANNER_API_ENDPOINT=staging-wrenchworks.sandbox.googleapis.com:443\n\n")
```

Comment:
It is a good idea to use constants for strings like this, so that if we change it in one place, it reflects everywhere.

Reply:
added constants